README.md

```diff
@@ -55,20 +55,43 @@ that you program with natural language instructions rather than code.
+system_prompt = "You are Richard Feynman. You will be given a difficult concept, and your task is to explain it back."
+feynman = SimpleBot(
+  system_prompt,
+  model_name="gpt-3.5-turbo"
+)
 ```
 
-Now, `feynman` is callable on any arbitrary chunk of text and will return a rephrasing of that text in Richard Feynman's style (or more accurately, according to the style prescribed by the prompt).
+For using GPT, you need to have the `OPENAI_API_KEY` environment variable configured. If you want to use `SimpleBot` with a local Ollama model, [check out this example](#using-simplebot-with-a-local-ollama-model)
+
+Now, `feynman` is callable on any arbitrary chunk of text and will return a rephrasing of that text in Richard Feynman's style (or more accurately, according to the style prescribed by the `system_prompt`).
 For example:
 
 ```python
-feynman("Enzyme function annotation is a fundamental challenge, and numerous computational tools have been developed. However, most of these tools cannot accurately predict functional annotations, such as enzyme commission (EC) number, for less-studied proteins or those with previously uncharacterized functions or multiple activities. We present a machine learning algorithm named CLEAN (contrastive learning–enabled enzyme annotation) to assign EC numbers to enzymes with better accuracy, reliability, and sensitivity compared with the state-of-the-art tool BLASTp. The contrastive learning framework empowers CLEAN to confidently (i) annotate understudied enzymes, (ii) correct mislabeled enzymes, and (iii) identify promiscuous enzymes with two or more EC numbers—functions that we demonstrate by systematic in silico and in vitro experiments. We anticipate that this tool will be widely used for predicting the functions of uncharacterized enzymes, thereby advancing many fields, such as genomics, synthetic biology, and biocatalysis.")
+prompt = """
+Enzyme function annotation is a fundamental challenge, and numerous computational tools have been developed.
+However, most of these tools cannot accurately predict functional annotations,
+such as enzyme commission (EC) number,
+for less-studied proteins or those with previously uncharacterized functions or multiple activities.
+We present a machine learning algorithm named CLEAN (contrastive learning–enabled enzyme annotation)
+to assign EC numbers to enzymes with better accuracy, reliability,
+and sensitivity compared with the state-of-the-art tool BLASTp.
+The contrastive learning framework empowers CLEAN to confidently (i) annotate understudied enzymes,
+(ii) correct mislabeled enzymes, and (iii) identify promiscuous enzymes with two or more EC numbers—functions
+that we demonstrate by systematic in silico and in vitro experiments.
+We anticipate that this tool will be widely used for predicting the functions of uncharacterized enzymes,
+thereby advancing many fields, such as genomics, synthetic biology, and biocatalysis.
+"""
+feynman(prompt)
```

Comment:
This is a lovely change, thank you so much! Makes it much easier to read in a no-line-wrap setting.